app/src/componentLibrary/autocompletes/singleAutocomplete/singleAutocomplete.jsx

```diff
@@ -86,6 +87,13 @@ export class SingleAutocomplete extends Component {
       ...rest,
     });
 
+  handleKeyDown = (event, isOpen, setHighlightedIndex) => {
+    if (event.keyCode === TAB_KEY_CODE && isOpen) {
+      event.preventDefault();
+      setHighlightedIndex(this.props.options.length);
+    }
```

Comment:
Could you please describe this block?

Reply:
So, if our input is focused and autocomplete menu is opened, I don't want to change the focus using the tab to next input. Because when input is blurred, autocomplete menu automatically closes. To elements at the same time can't be in focus. That is why we need just highlight +add new key or add new value button.